posts/2020-11-20-JNDI-gRPC-200012.adoc

```diff
@@ -212,10 +212,11 @@ Prior to this enhancement, Liberty only supported looking up enterprise beans us
 Without any additional configuration, legacy default JNDI names are now available for applications to use to lookup and/or inject enterprise beans. Also, rather than using the defaults, a custom name for each EJB may be specified in the `ibm-ejb-jar-bnd.xml` file (or `ibm-ejb-jar-bnd.xmi` file for EJB 2.x and 1.x modules). These new JNDI name options are available in addition to the existing specification required names.
 
 For EJB 3.x modules, the following defaults will be available if a custom name is not provided:
-Short form local interfaces and homes `ejblocal:<package.qualified.interface>`
-Short form remote interfaces and homes `<package.qualified.interface>`
-Long form local interfaces and homes `ejblocal:#<package.qualified.interface>`
-Long form remote interfaces and homes `ejb/#<package.qualified.interface>`
+
+Short form local interfaces and homes    `ejblocal:<package.qualified.interface>``
```

Comment:
This section is not formatting correctly; it ends up as one long wrapped line, and the wrong parts are highlighed as code.
I think part of the problem is an extra tick at the end of two lines; not sure what needs to be done to get each line to be separate rather than wrapping.

Reply:
Ah, okay I see it now. Should be fixed now. Draft site is rebuilding now.